sql/_05_plcsql/_01_testspec/_02_declaration/_07_local_proc/_05_common/answers/02_07_05-03_normal_return_with_value.answer

```diff
@@ -0,0 +1,13 @@
+===================================================
+Error:-1359
```

Comment:
The file name must be changed as shown below.

02_07_05-03_error_return_with_value.answer

Reply:
Yes, Thank you